[WPT] Fix import of css/css-cascade

mdubet opened the pull request:
#### f0c94e1f280ff6e8fc74f4bdc789857350caf08f
<pre>
[WPT] Fix import of css/css-cascade
<a href="https://rdar.apple.com/118203546">rdar://118203546</a>

Reviewed by Tim Nguyen.

Missing files from previous import <a href="https://commits.webkit.org/270319@main">https://commits.webkit.org/270319@main</a>

* LayoutTests/imported/w3c/web-platform-tests/css/css-cascade/idlharness-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/css-cascade-6.idl: Added.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/css-cascade.idl: Added.

Canonical link: <a href="https://commits.webkit.org/270531@main">https://commits.webkit.org/270531@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/29e561213557efbf84e6b540eef959ed6ac8b61b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/25706 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/4312 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/26990 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/27807 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/23545 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/6088 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/1747 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/23669 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/25955 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/3247 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/22156 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/28387 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/2874 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/23111 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/29183 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/23482 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/23479 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/27044 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/2869 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/1104 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/4250 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/6174 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/3319 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/3185 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
At some point we should just fully re-import the interfaces directory.

mdubet committed:
[WPT] Fix import of css/css-cascade
rdar://118203546

Reviewed by Tim Nguyen.

Missing files from previous import https://commits.webkit.org/270319@main

* LayoutTests/imported/w3c/web-platform-tests/css/css-cascade/idlharness-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/css-cascade-6.idl: Added.
* LayoutTests/imported/w3c/web-platform-tests/interfaces/css-cascade.idl: Added.

Canonical link: https://commits.webkit.org/270531@main
